media/css/custom.scss

```diff
@@ -471,3 +476,34 @@ summary {
 
 @import "faculty_structure";
 @import "honours_board";
+
+.relax-item {
+  width: 200px;
```

Comment:
Properties should be ordered background, float, height, margin-left, margin-top, width